pkg/nodeset/v1alpha1/types.go

```diff
@@ -27,6 +27,8 @@ type NodeClassResourceType string
 const (
 	NodeClassResourceFile      NodeClassResourceType = "File"
 	NodeClassResourceReference NodeClassResourceType = "Reference"
+
+	TPRNamespace = "default"
```

Comment:
please add a TODO that we will get rid of this once we port to the TPR successor.